sql/municipios.sql

```diff
@@ -5,5578 +5,5581 @@ CREATE TABLE municipios(
     longitude FLOAT(8) NOT NULL,
     capital BOOLEAN NOT NULL,
     codigo_uf INT NOT NULL,
-    PRIMARY KEY (codigo_ibge),
-    FOREIGN KEY (codigo_uf) REFERENCES estados (codigo_uf)
+    siafi_id INT NOT NULL,
+    ddd INT NOT NULL,
+    fuso_horario VARCHAR(50),
```

Comment:
Considerando que todos os municípios possuem um fuso horário (e não achei nenhum null nos dados atuais), acredito que faça sentido marcar essa coluna como `NOT NULL` também, o que você acha?

Reply:
O siafi_id não pode ser INT pois existem varios códigos do siafi que começam com 0 então ia ficar errado.
O fuso_horario pode ser um varchar de 32 caracteres não precisa ser de 50.
Recomendo dar uma olhada no SQL que eu fiz e publiquei na issue lá. https://pastebin.com/6w2Jn8qw